docs/pages/concepts/offloading/namespace-replication.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Namespace replication
+weight: 2
+---
+
+### The Liqo Namespace Model
+
+The Liqo resource replication model is composed of several concurring logics responsible for different kind of objects to replicate.
+One important example is represented by namespaces. 
+In particular, namespace replication across multiple clusters implements a key mechanism to extend seamleassly a cluster to some others.
+
+In the Liqo model, the namespace replication corresponds to a standard v1.Namespace and an associated [NamespaceOffloading](#) CR.
+The NamespaceOffloading resource allows to select the clusters which are the clusters where the application contained inside the namespace can be scheduled.
+
+In [a dedicated usage section](/usage/namespace_offloading), you can learn how to define and use a NamespaceOffloading resource activating the replication of the local namespace on peered clusters.
+
+In this section, you may discover how Liqo namespace model works under the hood, undestanding which resources and controllers are activated when a new namespaceOffloading is manipulated.
+
+#### Resources
+
+Liqo namespace replication involves instances of two kinds of CRDs:
+
+* First, the initial trigger is represented by the manipulation of a **NamespaceOffloading** (or the labelling of a namespace as explained in [the usage section](/usage/namespace_offloading)).
+  On the one hand, the *NamespaceOffloading* spec describes the properties of the replication, such as the name of the replicated namespaces.
+  On the other hand, the status collects the information about the actual status of remote namespaces (i.e. if the replication has succeeded).
+* Second, **NamespaceMap** resources contain the list of namespaces associated to a specific node.
+  The spec collects the list of desired namespaces for a specific remote cluster while the status updated information about their effective creation.
+  Each *NamespaceMap* can be alimented by several *NamespaceOffloading* instances that are targeting the same cluster for a remote namespace.
+  It is worth noting that the NamespaceMap status represent the source of truth to know the status of replicated namesapces on foreign clusters.
```

Comment:
Custom resources and resource fields must be both written in Pascal code: NamespaceMap, CurrentMapping, etc...
We do not use the backticks `NamespaceMap` because the actual Hugo theme, inserts a light yellow box with the copy and paste functionality, it is not like there in GitHub. 
For standard k8s words like status, spec, etc... we usually use lowercase letters as usual.